fiscalsim_us/variables/gov/states/unemployment_compensation.py

```diff
@@ -1,10 +1,10 @@
-from fiscalsim_us.model_api import *
-
-
-class unemployment_compensation(Variable):
-    value_type = float
-    entity = Person
-    label = "Unemployment compensation"
-    unit = USD
-    documentation = "Income from unemployment compensation programs."
-    definition_period = YEAR
+from fiscalsim_us.model_api import *
+
+
+class unemployment_compensation(Variable):
+    value_type = float
+    entity = Person
+    label = "unemployment compensation"

```

Comment:
@ss7886. I like the capital "Unemployment..." in the label. Let's change this back.